cli/src/cli.rs

```diff
@@ -956,7 +962,11 @@ pub fn parse_command(
             })
         }
         ("transfer", Some(matches)) => {
-            let lamports = lamports_of_sol(matches, "amount").unwrap();
+            let amount = if let Some(lamports) = lamports_of_sol(matches, "amount") {
```

Comment:
Good idea. I opted for `impl From<Option<u64>> for TransferAmount`, since the clap-utils input parsers seem to be doing an efficient job of reading matches and parsing SOL already.

Reply:
To that point, do you think I should go ahead and add the `ALL` option for the `create-*-account` commands at this time?

Reply:
It wouldn't hurt. Might even bring some clarity to how we can generalize the implementation.